config/consensus.go

```diff
@@ -422,10 +422,6 @@ var MaxEvalDeltaAccounts int
 // in a StateDelta, used for decoding purposes.
 var MaxStateDeltaKeys int
 
-// MaxLogCalls is the highest allowable log messages that may appear in
-// any version, used for decoding purposes. Never decrease this value.
-const MaxLogCalls = 32
-
 // MaxLogicSigMaxSize is the largest logical signature appear in any of the supported
 // protocols, used for decoding purposes.
 var MaxLogicSigMaxSize int
```

Comment:
I don't want to move this one variable out when all the nearby values are of the same variety - they are not consensus parameters, rather they are conservative esitmates of the largest that some encoded message part can be.  It's not just this variable that is like that.

Reply:
The problem with other values they are derived from consensus param, and this one is not because `MaxLogCalls` not in consensus. In fact it should not be even called `Max...`